docs/extensions.adoc

```diff
@@ -898,45 +898,16 @@ interceptor`. If you attach your interceptor to both of them and need a differen
 can of course build two different interceptors or add a parameter to your interceptor and create two instances, telling
 each at addition time whether it is attached to the method interceptor or the other one.
 
+[source,groovy,indent=0]
+----
+include::{sourcedir}/extension/InterceptorSpec.groovy[tag=interceptor-class]
+----
```

Comment:
Not sure, we'd need to replace the `I` in the examples

Reply:
Why do we need to replace it?
Or do I expect too much transfer ability from the reader that he knows that is a class implementing the listener interface?
Then we could keep it of course. :-D

Reply:
In the worst case, they could look at the doc-spec where this is coming from. :-)

Reply:
> Or do I expect too much transfer ability from the reader that he knows that is a class implementing the listener interface?

Yes, I think for new readers it would be confusing.

> In the worst case, they could look at the doc-spec where this is coming from. :-)

🤔 It would be nice to automatically have a link to the sources under the code snippets.

Reply:
Your wish is my command: https://github.com/spockframework/spock/pull/1904 :-)